packages/nodejs/.changesets/add-global-error-tracking-helper--improve-root-span-management-.md

```diff
@@ -0,0 +1,5 @@
+---
+bump: "minor"
+---
+
+Add global error tracking helper. Improve root span management.
```

Comment:
```suggestion
Add [`tracer.addError` helper function](https://docs.appsignal.com/nodejs/tracing/exception-handling.html) to set errors on the root span. If errors were set on child spans they were ignored.
```
It's a message for the user, so we should explain how the change impacts them. We should also update the linked page to explain how `tracer.addError` works.